src/crates/README.md

```diff
@@ -23,9 +26,23 @@ There is always new crates being released that can be useful in the development
 - [tokio](https://crates.io/crates/tokio) - asynchronous runtime
 - [wasm-pack](https://crates.io/crates/wasm-pack) - tool for building WebAssembly
 
+
+## Tables
+
+There are a few libraries for pretty-printing tables to the console:
+
+- [cli-table](https://crates.io/crates/cli-table)
+- [comfy-table](https://crates.io/crates/comfy-table)
+- [prettytable](https://crates.io/crates/prettytable-rs)
+- [tabled](https://crates.io/crates/tabled)
+- [term-table](https://crates.io/crates/term-table)
```

Comment:
So far, we only list out crates we directly use.  We might want to give more thought to how we would want t talk about relevant crates more generally.

Can you open an issue for us to discuss this before merging this?

Reply:
https://github.com/rust-cli/book/issues/158